src/main/java/net/homework/webapp/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package net.homework.webapp.service;
+
+import net.homework.webapp.exception.DepartmentNotFoundException;
+import net.homework.webapp.exception.EmployeeNotFoundException;
+import net.homework.webapp.model.Department;
+import net.homework.webapp.model.Employee;
+import net.homework.webapp.repository.DepartmentRepository;
+import net.homework.webapp.repository.EmployeeRepository;
+import org.springframework.stereotype.Service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService{
+
+    private final EmployeeRepository employeeRepository;
+
+    private final DepartmentRepository departmentRepository;
+
+    @Autowired
+    public EmployeeServiceImpl(EmployeeRepository employeeRepository,
+                               DepartmentRepository departmentRepository) {
+        this.employeeRepository = employeeRepository;
+        this.departmentRepository = departmentRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public List<Employee> getAllEmployees() {
+        List<Employee> employeeList = new ArrayList<>();
```

Comment:
why not 
return employeeRepository.findAll();

Reply:
fixed